mycodo/controllers/controller_trigger.py

```diff
@@ -274,11 +261,7 @@ def initialize_variables(self):
     def start_method(self, method_id):
         """ Instruct a method to start running """
         if method_id:
-            method = db_retrieve_table_daemon(Method, unique_id=method_id)
-            method_data = db_retrieve_table_daemon(MethodData)
-            method_data = method_data.filter(MethodData.method_id == method_id)
-            method_data_repeat = method_data.filter(
-                MethodData.duration_sec == 0).first()
+            method = load_method(method_id, self.logger)
```

Comment:
Previous implementation did database access to controller tables partly in the controller and partly in the method. The part that was done in method has been migrated to the controllers.